src/main/java/com/faforever/moderatorclient/mapstruct/GamePlayerStatsMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.faforever.moderatorclient.mapstruct;
+
+import com.faforever.moderatorclient.api.dto.GamePlayerStats;
+import com.faforever.moderatorclient.ui.domain.GamePlayerStatsFX;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.List;
+
+@Mapper(uses = {JavaFXMapper.class, GameMapper.class, PlayerMapper.class})
+public abstract class GamePlayerStatsMapper {
+
+    @Mapping(target = "beforeRating", ignore = true)
+    @Mapping(target = "afterRating", ignore = true)
+    @Mapping(target = "ratingChange", ignore = true)
+    public abstract GamePlayerStatsFX map(GamePlayerStats gamePlayerStats);
```

Comment:
We need to add the ignores for the backreferences on GameFX object (yes, GameFX is currently not used, but see later comment).